mysql-test/include/audit_log_events.inc

```diff
@@ -6,6 +6,8 @@ CREATE TABLE t1
              (c1 INT,
               c2 CHAR(20));
 INSERT INTO t1 VALUES (1,'a'),(2,'b'),(3,'c');
+INSERT INTO `t1` VALUES (4,NULL);
+INSERT INTO `t1` VALUES (5,'541S?');
```

Comment:
Probably wouldn't hurt to test the escaped character set exhaustively.

Reply:
The tests currently are far from exhaustive (for example, the actual xml or json output isn't tested, only to the point that it is valid), these statements here only verify the SQL statements that were executed when creating the database.

So unless I also add some actual output verification, exhaustive testing wouldn't accomplish much more. Should I do that?

Reply:
It's just 31 characters - if sorted in input, recording the manual outputs of \01 \02 ... etc would be enough